var/spack/repos/builtin/packages/g2c/package.py

```diff
@@ -46,10 +48,18 @@ class G2c(CMakePackage):
         description="Build and install some utility programs",
         when="@develop",
     )
+    variant(
+        "build_g2c",
```

Comment:
I would call this variant `new_api` or something of the sort. `build_g2c` might give folks the wrong impression that this variant installs from a binary instead of from source.

Reply:
OK, sorry I didn't see this. Can we rename this in the spack file, because it's already been released.

We can call it v2_api if that helps.